### ISSUE TITLE
Screen reader announcement for "No search results" message

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#25629]
## Issue Description
When a search on Facility Locator returns no results, a message is displayed to inform the user. 

<details> 
<summary> Screenshot </summary>

![image](https://user-images.githubusercontent.com/55411834/121247593-a7f17680-c85f-11eb-9fc9-1eb39cf078b5.png)

</details>

When no results are found, an error message is displayed but the usual search results header is absent (ie: Results for "VA health", "All VA health services" near "Spokane, Washington 99224") 

Therefore, the message is not announced to screen readers, leaving the user wondering what they did wrong and how to fix it. 

This issue was discovered during the Screen reader accessibility research pilot and observed during Veteran sessions

**Consider:** 
Because the usual search results header is an H2, displaying it for all searches - successful or not - would ensure it is announced to screen readers 
- "No results for "VA health", "All VA health services" near "Spokane, Washington 99224" , for example

## Tasks
- [x] Show the search results subheader with "No results found for ...<search criteria>".
- [x] Delete first sentence of the help text "We didn't find any facilities near you."
- [x] Update / add tests

## Acceptance Criteria
- [ ] All Veterans are informed when a search returns no results.
- [ ] Screen reader announces no results have been returned
- [ ] Search results header receives focus 
- [ ] The "Can't find what you're looking for?" link appears beneath the map after every search
- [ ] This issue to be validated by Angela/Josh

**Comment by mmiddaugh** [comment]
@joshkimux @AngelaFowler82 when you have a moment, this issue is ready for validation

**Comment by AngelaFowler82** [comment]
JAWS and NVDA announce “no search results,” followed by the specified criteria. Focus is placed at the h2. I think we can close this.
